proto/tabletmanagerdata.proto

```diff
@@ -717,8 +717,18 @@ message ResetSequencesResponse {
 
 message CheckThrottlerRequest {
   string app_name = 1;
+
+  string scope = 2;
+  // SkipRequestHeartbeats ensures this check does not renew heartbeat lease
+  bool skip_request_heartbeats = 3;
+  // OKIfNotExists asks the throttler to return OK even if the metric does not exist
+  bool ok_if_not_exists = 4;
+  // MultiMetricsEnabled is always set to "true" and is how a multi-metrics enabled replica
+  // throttler knows its being probed by a multi-metrics enabled primary vttablet.
+  bool multi_metrics_enabled = 5;
```

Comment:
If it's true, why do we need it? The zero value will be default so we'll have to be sure we're setting it to true everywhere. When should it be false? Is it related to backwards compatibility, or mixed tablet versions?

Reply:
> Is it related to backwards compatibility,
Yes! As per the comment to this variable which I could probably rephrase better.

> ... and is how a multi-metrics enabled replica
>  // throttler knows its being probed by a multi-metrics enabled primary vttablet.

Meaning a "new" replica would respond like an "old" replica would, if this boolean is `false`, because that indicates an "old" primary.

Reply:
The flag will be completely removed in v22.